.editorconfig

```diff
@@ -0,0 +1,21 @@
+# EditorConfig helps developers define and maintain consistent coding styles between different editors and IDEs.
+# See http://editorconfig.org for details.
+
+# This is the top-most .editorconfig file; do not search in parent directories.
+root = true
+
+# All files.
+
+[*]
+indent_style = space
+indent_size = 2
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+end_of_line = lf
+max_line_length = null
+
+
+# Markdown.
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
I would rather not add this

I've never seen it before, some of it (indent, indent size, whitespace, newline, end of line, max length) is redundant of other tools in place, and I just don't see a benefit since we don't care what any individual writes since it will be overwritten during commit anyway

Reply:
> overwritten during commit anyway
via prettier

Reply:
> I've never seen it before

It's [standard](https://github.com/nodejs/node/blob/master/.editorconfig) [in](https://github.com/eslint/eslint/blob/master/.editorconfig) [many](https://github.com/facebook/jest/blob/master/.editorconfig) [javascript](https://github.com/babel/babel/blob/master/.editorconfig) [projects](https://github.com/okonet/lint-staged/blob/master/.editorconfig) and it configures editors uniformly and correctly up front rather than waiting for the linter to correct, but if you're not comfortable with it, no problem.